sections/gov/components/List/CouncilBoard.tsx

```diff
@@ -25,7 +24,6 @@ const CouncilBoard: React.FC = () => {
 
 	return (
 		<StyledCard>
-			<Img width={'100%'} src={SpartanCouncilNFT} />
```

Comment:
you don't think we should show the ```gif``` of the nft anymore?

Reply:
yea I temp removed because I was having build error, but its been re-added again now.